src/Protocol/PortableContact.php

```diff
@@ -1670,7 +1673,7 @@ public static function discoverSingleServer($id)
 				self::discoverServer($data, 2);
 			}
 
-			if (Config::get('system', 'poco_discovery') > 1) {
+			if (Config::get('system', 'poco_discovery') >= self::USERS_GCONTACTS) {
```

Comment:
Wouldn't be a binary "has" better? It seems like `>=` is used instead of "has" GCONTACTS and GCONTACTS_FALLBACK